### DISCUSSION TITLE
Screenshot competition! 📸

### DISCUSSION THREAD

**Discussion by ErikBjare** [ActivityWatch/activitywatch#741] [General]
We're in need of beta testing & more screenshots for ActivityWatch, and we want your help!

## To participate

 1. Download & install/upgrade ActivityWatch (latest version as of writing is v0.12.0b2)
 2. Hard-refresh the web UI (to make sure your browser is not holding on to a stale cache of the UI)
 3. Take some nice screenshots, and upload them here!
     - You can optionally also share them on:
         - [Discord](https://discord.gg/vDskV9q) (the `#screenshots` channel)
         - Twitter (remember to tag [@ActivityWatchIt](https://twitter.com/ActivityWatchIt))
     - Submissions might be shared on the website, Twitter, and other social media networks. 
     - Tip: If you want to hide something sensitive from the screenshot, you can first use dev tools to edit the page.

## Examples

 - The current screenshots on [the website](https://activitywatch.net/) and in the [README](https://github.com/ActivityWatch/activitywatch#screenshots)
 - This screenshot of ActivityWatch data imported to Metabase, by @SqrtMinusOne: https://twitter.com/ActivityWatchIt/status/1522126015082151936

## Rules

 - Deadline is ~~2022-06-05~~
   - We will be continuing to allow new submissions, but prizes will be announced ~~shortly~~ *when we have them*! (sorry for the wait!)
 - Screenshot can be anything AW related (not just the web UI)
 - If possible, try to include the version number at the bottom of the UI
 - High-quality PNG preferred
 - One reply per user (but multiple images per reply welcome!)

## Prizes

There will be a mystery prize to the top 3 submissions! 

## Links

Twitter announcement: https://twitter.com/ActivityWatchIt/status/1526123764794245120

**Comment by ClementineAccount** [comment]
Might submit something else later if inspiration hits
![screenshot](https://user-images.githubusercontent.com/26779639/168577134-12c9a006-b510-4466-952f-ef56d249469c.gif)
 
---

Another interpretation of the theme.
![image](https://user-images.githubusercontent.com/26779639/168577810-9a8fc6ed-e9fb-4cea-8e9d-3a7867a03157.png)

**Comment by BelKed** [comment]
I've been using ActivityWatch for about a month now and I'm pleased with it.
As you can see from the screenshot, I really love the dark theme :D

![Screen Shot 2022-05-16 at 22 20 59](https://user-images.githubusercontent.com/66956532/168841434-e2eace81-f72f-4e4f-a39a-d629d4f0496b.png)

---

The whole website (with version):

![Screen Shot 2022-05-16 at 22 26 05](https://user-images.githubusercontent.com/66956532/168841448-cea26bee-0607-4b22-8cb4-72904dd32f18.png)

**Comment by ErikBjare** [comment]
By Nitroplex in the Discord

![image](https://user-images.githubusercontent.com/1405370/172569760-7b4fabc6-ded4-4a31-b385-cc757631836d.png)
![image](https://user-images.githubusercontent.com/1405370/172569793-cb9e310b-bd7a-4612-8d59-f2fdd95d410f.png)

**Comment by ErikBjare** [comment]
By Erik (Thelife#6292) in the Discord:

>Hey Erik, It's Erik. I did an entire photoshoot. Feel free to check it out (If you want anything specific shots to be taken, let me know! I want to give back!): Here are some samples 
>Appologies for all the 7z files, my internet is acting up and I can't seem to upload to google drive

![image](https://user-images.githubusercontent.com/1405370/172569986-e1810df9-203d-4d55-8f32-36c2f3efb539.png)
![image](https://user-images.githubusercontent.com/1405370/172570077-f180bb31-bacd-472f-93d4-ecbd75ecb437.png)

**Reply by erikmakela** [reply]
Didn't realize I could post them all to Github. Here are all photo's I took. -Erik

![Screen Shot 2022-05-16 at 17 53 57-fullpage](https://user-images.githubusercontent.com/91350035/175657492-af3f9c82-6ae5-4845-aba4-fb5c6b339a49.png)
![Screen Shot 2022-05-16 at 17 54 25-fullpage](https://user-images.githubusercontent.com/91350035/175657493-dc8d6d1f-4f24-420c-860d-97a7573a00c0.png)
![Screen Shot 2022-05-16 at 18 02 38-fullpage](https://user-images.githubusercontent.com/91350035/175657495-697df310-1b79-4c86-b255-acfbddc7
![Screen Shot 2022-05-16 at 17 27 46-fullpage](https://user-images.githubusercontent.com/91350035/175657568-caa6f789-5a72-4b8a-8689-62ae1facae57.png)
![Screen Shot 2022-05-16 at 17 29 23-fullpage](https://user-images.githubusercontent.com/91350035/175657569-bfd99e29-20a4-4d60-97ad-644cc4e7b765.png)
![Screen Shot 2022-05-16 at 17 30 51-fullpage](https://user-images.githubusercontent.com/91350035/175657571-2cb22164-0e85-44a4-908a-924baeaa0957.png)
![Screen Shot 2022-05-16 at 17 31 12-fullpage](https://user-images.githubusercontent.com/91350035/175657575-dda1a19d-9393-45e6-9c6d-214d9f490f07.png)
![Screen Shot 2022-05-16 at 17 31 42-fullpage](https://user-images.githubusercontent.com/91350035/175657576-eca8d2ec-85fa-4e6b-bed1-6a35981bde2e.png)
![Screen Shot 2022-05-16 at 17 31 55-fullpage](https://user-images.githubusercontent.com/91350035/175657583-793e68f9-3534-47e1-bb91-659b1059bc55.png)
![Screen Shot 2022-05-16 at 17 32 21-fullpage](https://user-images.githubusercontent.com/91350035/175657587-291d2672-1f62-4680-93f7-c3fc5db1c5e2.png)
![Screen Shot 2022-05-16 at 17 32 31-fullpage](https://user-images.githubusercontent.com/91350035/175657590-b6fa3888-a394-49ca-bb7c-d9946c9e3fb0.png)
![Screen Shot 2022-05-16 at 17 33 21-fullpage](https://user-images.githubusercontent.com/91350035/175657592-08616981-e871-4914-bc91-7bd028625f41.png)
![Screen Shot 2022-05-16 at 17 33 44-fullpage](https://user-images.githubusercontent.com/91350035/175657597-cb607e64-06b2-4df2-bcbd-2c66eeac449e.png)
![Screen Shot 2022-05-16 at 17 34 22-fullpage](https://user-images.githubusercontent.com/91350035/175657599-a5da8cc0-9fba-428c-9f27-25148a9f9102.png)
![Screen Shot 2022-05-16 at 17 34 43-fullpage](https://user-images.githubusercontent.com/91350035/175657603-55f17870-a48b-4d66-8da4-6636cdd7ac4a.png)
![Screen Shot 2022-05-16 at 17 35 19-fullpage](https://user-images.githubusercontent.com/91350035/175657604-2695e682-4b41-4aa5-b37a-84bb71b8cf3f.png)
![Screen Shot 2022-05-16 at 17 35 42-fullpage](https://user-images.githubusercontent.com/91350035/175657606-9acd4f06-f9fe-455f-b46a-d16fdfbc9014.png)
![Screen Shot 2022-05-16 at 17 37 48-fullpage](https://user-images.githubusercontent.com/91350035/175657607-b5cdd963-616d-45fd-98b7-3c5a933fee52.png)
![Screen Shot 2022-05-16 at 17 38 10-fullpage](https://user-images.githubusercontent.com/91350035/175657609-36182078-1ed6-46dd-9e3e-08a69c457c60.png)
![Screen Shot 2022-05-16 at 18 04 38-fullpage](https://user-images.githubusercontent.com/91350035/175657610-90ee477c-db24-4a11-85c5-689f2cea35fc.png)
![Screen Shot 2022-05-16 at 18 05 08-fullpage](https://user-images.githubusercontent.com/91350035/175657612-00aec167-e4f4-4c0a-b393-39cad60dc774.png)
![Screen Shot 2022-05-16 at 17 49 07-fullpage](https://user-images.githubusercontent.com/91350035/175657614-c0b122ff-df26-44a1-a0fc-7751fe0d0347.png)
![Screen Shot 2022-05-16 at 17 48 00-fullpage](https://user-images.githubusercontent.com/91350035/175657616-c6b8a406-6e8f-48e2-a1a4-5ae4c414674c.png)
![Screen Shot 2022-05-16 at 18 07 00-fullpage](https://user-images.githubusercontent.com/91350035/175657619-1020c65c-e1f3-4ed7-a85b-82b69e89781e.png)
![Screen Shot 2022-05-16 at 18 06 31-fullpage](https://user-images.githubusercontent.com/91350035/175657622-59ece09b-75fa-4cd2-809a-69fc840e3336.png)
![Screen Shot 2022-05-16 at 17 39 15-fullpage](https://user-images.githubusercontent.com/91350035/175657626-a793261d-f441-4cc9-a842-6aefa40b1123.png)
![Screen Shot 2022-05-16 at 17 38 51-fullpage](https://user-images.githubusercontent.com/91350035/175657627-cbf72b35-22db-4c90-be72-192738b6c230.png)
891a.png)
![Screen Shot 2022-05-16 at 18 02 55-fullpage](https://user-images.githubusercontent.com/91350035/175657499-62362b93-af26-4dfe-acf5-1f51f2e30ac6.png)
![Screen Shot 2022-05-16 at 18 03 06-fullpage](https://user-images.githubusercontent.com/91350035/175657503-329cee9d-a891-4343-9bc0-38704ec554cb.png)
![Screen Shot 2022-05-16 at 18 03 19-fullpage](https://user-images.githubusercontent.com/91350035/175657504-b1bf9040-9235-4eeb-914d-0f9efd767e98.png)
![Screen Shot 2022-05-16 at 17 53 08-fullpage](https://user-images.githubusercontent.com/91350035/175657505-3d6c5e9b-9e2a-4bc4-b700-2e93a7cd2524.png)
![Screen Shot 2022-05-16 at 17 53 21-fullpage](https://user-images.githubusercontent.com/91350035/175657509-ee017468-40c0-4333-b994-3c91dfd79d30.png)
![Screen Shot 2022-05-16 at 17 40 19-fullpage](https://user-images.githubusercontent.com/91350035/175657510-ab987689-1bb8-4da9-a6b9-3778a8a766cf.png)
![Screen Shot 2022-05-16 at 17 39 57-fullpage](https://user-images.githubusercontent.com/91350035/175657515-7ea09f7d-b3b0-4b23-8e94-8297ea9b975a.png)
![Screen Shot 2022-05-16 at 18 01 33-fullpage](https://user-images.githubusercontent.com/91350035/175657517-30b3d7de-b0b0-4cc1-b4fb-87a901737482.png)
![Screen Shot 2022-05-16 at 17 57 12-fullpage](https://user-images.githubusercontent.com/91350035/175657520-5c913cfc-cfaa-4ac4-8bcc-903948109a07.png)
![Screen Shot 2022-05-16 at 17 58 50-fullpage](https://user-images.githubusercontent.com/91350035/175657525-4985cd82-99b7-4bb2-9132-1e0868bc81a7.png)
![Screen Shot 2022-05-16 at 18 00 30-fullpage](https://user-images.githubusercontent.com/91350035/175657528-433016f6-3cfa-40ff-92a9-385d838576db.png)
![Screen Shot 2022-05-16 at 17 13 00-fullpage](https://user-images.githubusercontent.com/91350035/175657531-88caaf91-5396-4995-9a5e-f1e9c5e50e11.png)
![Screen Shot 2022-05-16 at 17 14 07-fullpage](https://user-images.githubusercontent.com/91350035/175657539-01bddc95-ae33-4523-b107-415879cf3eb7.png)
![Screen Shot 2022-05-16 at 17 15 49-fullpage](https://user-images.githubusercontent.com/91350035/175657545-09d4ed8f-124b-433a-8761-6fdf0db10d9f.png)
![Screen Shot 2022-05-16 at 17 17 44-fullpage](https://user-images.githubusercontent.com/91350035/175657550-20ddfc77-79f7-407c-8bf1-b29588a16743.png)
![Screen Shot 2022-05-16 at 17 21 54](https://user-images.githubusercontent.com/91350035/175657553-8cb3208c-ac7a-4966-8529-7d278550b3f7.png)
![Screen Shot 2022-05-16 at 17 23 54](https://user-images.githubusercontent.com/91350035/175657555-ee832d4f-ddbb-458c-b946-cb0041c26e2e.png)
![Screen Shot 2022-05-16 at 17 25 38-fullpage](https://user-images.githubusercontent.com/91350035/175657557-479b4258-7c82-494b-abe5-1ea571a998ed.png)
![Screen Shot 2022-05-16 at 17 26 34-fullpage](https://user-images.githubusercontent.com/91350035/175657562-eeae6858-2ba1-4023-8f78-a1b2bbf35b2e.png)

**Comment by ErikBjare** [comment]
By Ser#5766 in the Discord:

![image](https://user-images.githubusercontent.com/1405370/172570253-9b46d2fe-7d9d-4ac2-8b40-b04e7da3ad92.png)

**Comment by ErikBjare** [comment]
By maholli#4290 in the Discord:

>Typical month for me. Forever plagued by uncategorized :/

![image](https://user-images.githubusercontent.com/1405370/172570365-39d8e37b-08bf-4c5d-a105-ce06da9e68a1.png)
